src/input-range-select/input-range-select.tsx

```diff
@@ -134,6 +136,12 @@ export const InputRangeSelect = <T, V>({
         }
     };
 
+    const handleClear = (event: { stopPropagation: () => void }) => {
+        setSelectedFromValue(undefined);
+        setSelectedToValue(undefined);
```

Comment:
you should also fire the `onSelectOption` callback prop here to indicate that the selection has been updated